src/panels/lovelace/components/hui-graph-base.ts

```diff
@@ -31,18 +31,18 @@ export class HuiGraphBase extends LitElement {
                 d="${this._path} L 500, 100 L 0, 100 z"
               />
             </mask>
-            <rect height="100%" width="100%" id="fill-rect" fill="var(--accent-color)" mask="url(#fill)"></rect>
+            <rect height="100%" width="100%" id="fill-rect" fill="var(--graph-color, var(--accent-color))" mask="url(#fill)"></rect>
```

Comment:
not sure if `graph-color` is the right name, as it will only be used for the svg graph and not all the other graphs we have.

Reply:
Oh, though this was the `graph-base`. What's the source of the other graphs? Are they auto-colored?

Reply:
Do you test your changes?

Reply:
No, sometimes I should see how to do it.

Reply:
Okay, does this work, or is it a bad implementation?

Reply:
This looks bad. We are only modifying the first color?

Reply:
Well that way you can tell them apart, and it's still customizable.

Reply:
We won't accept any changes if you don't test them. Please set up a proper dev env first.